elasticsearch/elasticsearch.d.ts

```diff
@@ -125,6 +129,17 @@ declare module Elasticsearch {
         body: string | any;
     }
 
+    export interface IndicesPutMappingParams extends GenericParams {
+        timeout?: Date | number;
+        masterTimeout?: Date | number;
+        ignoreUnavailable?: boolean;
+        allowNoIndices?: boolean;
+        expandWildcards?: string;
```

Comment:
Changed. I changed expandWildcards in `SuggestParams` and `IndicesGetAliasParams` definitions too.